src/index.js

```diff
@@ -25,40 +25,52 @@ const upcomingTripTemplate = document.querySelector('.future-trips-template')
 
 const pageLoad = () => {
   let rando = (Math.ceil(Math.random() * 49))
-  getTraveler(rando)
-  .then(traveler => traveler = new Traveler(traveler.id, traveler.name, traveler.travelerType))
-  .then(traveler => dashboardGreeting.innerText = "Hello, " + (traveler.returnFirstNameLastInitial()))
 
-  getTrips()
-    .then(trips => trips = trips.map(trip => { 
-    return new Trip(trip)}))
-    .then(trips => trips.filter(trip => {
+  const travelerResults = getTraveler(rando)
+    .then(traveler => traveler = new Traveler(traveler.id, traveler.name, traveler.travelerType))
+    .then(traveler => dashboardGreeting.innerText = "Hello, " + (traveler.returnFirstNameLastInitial()))
+
+  const tripsResults = getTrips()
+
+  const placeResults = getDestinations()
+    .then(destinations => destinations = destinations.map(destination => {
+      return new Destination(destination)
+    }))
+    
+
+  Promise.all([travelerResults, tripsResults, placeResults])
+    .then(values => values[1].map(trip => {
```

Comment:
- I, personally, have had a lot of trouble mapping through everything I need in a `Promise.all()`
- I'd consider creating global variables, then _pushing_ to them from inside the `Promise.all()`
  - this is more consistent than mapping and re-assigning